diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/40users mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/40users
--- e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/40users	2006-03-13 20:14:55.936805983 +1100
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/40users	2006-03-13 20:13:38.369965634 +1100
@@ -8,10 +8,14 @@
     for my $user ( $adb->users )
     {
 	my %props = $user->props;
+	$props{AllowRSSH} ||= 'unknown';
 
 	next unless ($props{PasswordSet} eq 'yes');
 
-	next unless ($props{AllowRSSH} eq 'yes');
+	next if ($props{AllowRSSH} eq 'no');
+
+	next unless ($props{AllowRSSH} eq 'yes' or
+		$props{VPNClientAccess} eq 'yes');
 
 	$OUT .= "user = " . $user->key . ":022:11111:";
     }
