diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.12.0/root/etc/e-smith/templates/etc/ssh/sshd_config/10Port mezzanine_patched_e-smith-openssh-1.12.0/root/etc/e-smith/templates/etc/ssh/sshd_config/10Port
--- e-smith-openssh-1.12.0/root/etc/e-smith/templates/etc/ssh/sshd_config/10Port	2002-02-15 07:51:15.000000000 +1100
+++ mezzanine_patched_e-smith-openssh-1.12.0/root/etc/e-smith/templates/etc/ssh/sshd_config/10Port	2006-07-25 19:08:48.110898816 +1000
@@ -1 +1,5 @@
-Port 22
+{
+    my $ssh_port = $sshd{'TCPPort'} || 22;
+
+    $OUT = "Port $ssh_port";
+}
