diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/createlinks mezzanine_patched_e-smith-openssh-1.11.0/createlinks
--- e-smith-openssh-1.11.0/createlinks	2006-01-23 16:25:30.713468200 +1100
+++ mezzanine_patched_e-smith-openssh-1.11.0/createlinks	2006-01-23 16:22:16.036339833 +1100
@@ -4,6 +4,7 @@
 use esmith::Build::CreateLinks qw(:all);
 
 foreach (qw(
+    /etc/rssh.conf
     /etc/ssh/sshd_config
     /root/.ssh/config
 	))
diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowCVS mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowCVS
--- e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowCVS	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowCVS	2006-01-23 16:25:20.555235618 +1100
@@ -0,0 +1 @@
+no
diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowRDIST mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowRDIST
--- e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowRDIST	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowRDIST	2006-01-23 16:25:20.555235618 +1100
@@ -0,0 +1 @@
+no
diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowRSYNC mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowRSYNC
--- e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowRSYNC	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowRSYNC	2006-01-23 16:25:20.555235618 +1100
@@ -0,0 +1 @@
+no
diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowSCP mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowSCP
--- e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowSCP	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowSCP	2006-01-23 16:22:54.739605899 +1100
@@ -0,0 +1 @@
+yes
diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowSFTP mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowSFTP
--- e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowSFTP	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/db/configuration/defaults/sshd/AllowSFTP	2006-01-23 16:22:54.739605899 +1100
@@ -0,0 +1 @@
+yes
diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/10logfacility mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/10logfacility
--- e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/10logfacility	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/10logfacility	2006-01-23 16:12:48.997998132 +1100
@@ -0,0 +1 @@
+logfacility = LOG_USER 
diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/10umask mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/10umask
--- e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/10umask	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/10umask	2006-01-23 16:12:53.757170090 +1100
@@ -0,0 +1 @@
+umask = 022
diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowcvs mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowcvs
--- e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowcvs	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowcvs	2006-01-23 16:20:18.325820140 +1100
@@ -0,0 +1,3 @@
+{
+    return "allowcvs" if ($sshd{AllowCVS} eq "yes");
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowrdist mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowrdist
--- e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowrdist	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowrdist	2006-01-23 16:20:35.782782825 +1100
@@ -0,0 +1,3 @@
+{
+    return "allowrdist" if ($sshd{AllowRDIST} eq "yes");
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowrsync mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowrsync
--- e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowrsync	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowrsync	2006-01-23 16:20:50.262263558 +1100
@@ -0,0 +1,3 @@
+{
+    return "allowrsync" if ($sshd{AllowRSYNC} eq "yes");
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowscp mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowscp
--- e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowscp	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowscp	2006-01-23 16:20:52.451882589 +1100
@@ -0,0 +1,3 @@
+{
+    return "allowscp" if ($sshd{AllowSCP} eq "yes");
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowsftp mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowsftp
--- e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowsftp	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/rssh.conf/20allowsftp	2006-01-23 16:19:41.791176758 +1100
@@ -0,0 +1,3 @@
+{
+    return "allowsftp" if ($sshd{AllowSFTP} eq "yes");
+}
