diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/ssh/sshd_config/50SubsystemSftp mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/ssh/sshd_config/50SubsystemSftp
--- e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/ssh/sshd_config/50SubsystemSftp	2006-03-02 18:29:19.336170056 +1100
+++ mezzanine_patched_e-smith-openssh-1.11.0/root/etc/e-smith/templates/etc/ssh/sshd_config/50SubsystemSftp	2006-03-02 18:28:59.456628873 +1100
@@ -6,6 +6,6 @@
     my $SubsystemSftp = $sshd{"SubsystemSftp"} || "yes";
 
     $OUT = ($SubsystemSftp eq "yes") ? 
-	"Subsystem      sftp    /usr/libexec/openssh/sftp-server" : 
+	"Subsystem      sftp    /usr/libexec/sftp-server" : 
 	"# The sftp Subsystem is disabled in the configuration database";
 }
