diff -Nur -x '*.orig' -x '*.rej' e-smith-openssh-1.11.0/createlinks mezzanine_patched_e-smith-openssh-1.11.0/createlinks
--- e-smith-openssh-1.11.0/createlinks	2006-03-13 20:23:48.366370507 +1100
+++ mezzanine_patched_e-smith-openssh-1.11.0/createlinks	2006-03-13 20:23:33.639874231 +1100
@@ -4,7 +4,6 @@
 use esmith::Build::CreateLinks qw(:all);
 
 foreach (qw(
-    /etc/rssh.conf
     /etc/ssh/sshd_config
     /root/.ssh/config
 	))
@@ -16,6 +15,21 @@
 	));
 }
 
+foreach (qw(
+    /etc/rssh.conf
+	))
+{
+    templates2events("$_", qw(
+	bootstrap-console-save
+	password-modify
+	remoteaccess-update
+	user-lock
+	user-create
+	user-delete
+	user-modify
+	));
+}
+
 foreach my $event (
     "console-save",
     "bootstrap-console-save",
